fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/TypeRdfContext.java

```diff
@@ -58,71 +61,36 @@ public TypeRdfContext(final FedoraResource resource,
 
         //include rdf:type for primaryType, mixins, and their supertypes
         concatRdfTypes();
+        if (resource() instanceof NonRdfSource) {
+            // gather versionability info from the parent
+            if (resource().getNode().getParent().isNodeType("mix:versionable")) {
```

Comment:
Thanks for the clarification about both the description and the NonRdfSource being versioned. Your code makes it look like the triple stating that the NonRdfSource is versioned is derived from whether the NonRdfSource's parent is versioned. Am I misreading this:
https://github.com/fcrepo4/fcrepo4/pull/844/files#diff-e7befd57ec900402b6ffad300b697af1R66

As far as any desire to have one or the other of the description or NonRdfSource versioned and not the other, I have none. I am sorry if I gave an indication otherwise.